wasm/dotnet/src/dotnet/Benchmarks/BenchTask.cs

```diff
@@ -0,0 +1,125 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+public abstract class BenchTask
+{
+    public abstract string Name { get; }
+    readonly List<Result> results = new();
+    public Regex pattern;
+
+    public virtual bool BrowserOnly => false;
+
+    public virtual int BatchSize => 3;
```

Comment:
See comment on `Program.cs`, change to 100 to have a more sensible proportion between the String/JSON/Exception vs. Raytracer workloads.